Big-Bumper-Bash/Assets/UI/CarChoiceScript.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class CarChoiceScript : MonoBehaviour
+{
+    public Button defaultSelected;
+    public GameObject car1Button, car2Button, backButton;
+    public GameObject backMenu;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+    }
+
+    private void OnEnable()
+    {
+        defaultSelected.GetComponent<Button>().Select();
+    }
+
+    public void PressCar(string car = "no car chosen")
```

Comment:
This method certainly doesn't press a car. Correct me if I'm mistaken but they're called when a button is pressed. If that's the case then a better naming convention for them is `OnXyzPressed`. This applies to all the methods across the different screen. Also if the cars are in a determined set, it's better to represent them using an enum.
```suggestion
    public void OnChooseCarButtonPressed(string car = "no car chosen")
```